AppCenter/AppCenter/Internals/AppCenter+Internal.h

```diff
@@ -1,21 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why delete this header file?

Reply:
This should not be deleted, I was trying to move it to the right place in the module and apparently it didn't relink, we'll fix it